### ISSUE TITLE
More command line options

### ISSUE THREAD

**Issue by t00m** [t00m/KB4IT#29]
Allow users to pass more command line options.

For example:

- Force a new and clean compilation
- Create a site only for a list of specific keys/values
- etc...

**Comment by t00m** [comment]
Fix -f | --force option with commit 0f4d1f3e950254a19ba46b767fda066b0dc2205d

**Comment by t00m** [comment]
Current arguments
  
```
  -h, --help            show this help message and exit
  -reset                Reset environment
  -force                Force a clean compilation
  -theme THEME          Specify theme. Otherwise, it uses the default one
  -source SOURCE_PATH   Source directory with asciidoctor source files
  -target TARGET_PATH   Target directory
  -sort SORT_ATTRIBUTE  Choose another attribute for sorting instead the default timestamp
  -log LOGLEVEL         Increase output verbosity
  -version              show program's version number and exit
```

Enough for 0.8
